release-notes.txt

```diff
@@ -1,3 +1,567 @@
+RIOT-2021.07 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release
+==================
+
+The 2021.07 release includes:
+
+- Many improvements to ztimer usage
+- Added nucleo-wl55jc and Seeeduino XIAO boards
+- stm32f1 kconfig support
+- BadAlloc fix for malloc thread safety
+- Fixed multiplication overflow in calloc, a.k.a. BadAlloc, for all supported
+platforms
```

Comment:
```suggestion
- Fixed multiplication overflow in calloc, a.k.a. BadAlloc, for all supported
  platforms
```

Reply:
done

Reply:
Hmm, looks like the previous state got push again...